src/app/projects/projects.route.js

```diff
@@ -13,11 +13,19 @@
         templateUrl: 'app/projects/projects.html',
         controller: 'ProjectsCtrl',
         controllerAs: 'projects',
-        resolve: { $projects: getProjects }
+        resolve: {
+          isAuthenticatedWithFirebase: isAuthenticatedWithFirebase,
```

Comment:
Instead of using resolve this might be better placed with the google auth stuff..?

https://github.com/jgeewax/phoenix/blob/master/src/app/index.run.js#L25